bot/exts/info/code_snippets.py

```diff
@@ -18,22 +18,21 @@
 )
 
 GITHUB_GIST_RE = re.compile(
-    r'https://gist\.github\.com/([^/]+)/(?P<gist_id>[^\W_]+)/*'
-    r'(?P<revision>[^\W_]*)/*#file-(?P<file_path>\S+?)'
+    r'https://gist\.github\.com/([^/]+)/(?P<gist_id>[a-zA-Z0-9]+)/*'
```

Comment:
This should use the same regex for the username that the normal GitHub regex does. The current regex you use is too permissive.